docs/semgrep-code/policies.md

```diff
@@ -131,4 +108,45 @@ To disable a rule, follow these steps:
 
 You can also select individual rules under the **Mode** column and disable them one by one.
 
+## Policies filter reference
+
+This section explains columns in the rule pane of the Policies page in detail:
+
+| Filter | Description | Examples or possible values |
+| -------  | ------ | ------ |
+| Rule name  | Name of the rule that Semgrep Code uses for scanning.       | [<i class="fas fa-external-link fa-xs"></i> `docs-print-to-logger`](https://semgrep.dev/playground/s/KPzL)       |
+| Severity  | The higher the severity, the more critical the issues are that a rule detects.      |  <ul><li>High</li><li>Medium</li><li>Low</li></ul>      |
+| Confidence  | Indicates confidence of the rule to detect true positives.      |  <ul><li>High</li><li>Medium</li><li>Low</li></ul>      |
+| Source  | Indicates the origin of a rule.       | <ul><li><strong>Pro:</strong> Authored by Semgrep with cross-file (interfile), and cross-function (interprocedural) analysis capabilities providing you with enhanced scan accuracy. For more information, see <a href="/semgrep-code/pro-rules/"><i class="fa-regular fa-file-lines"></i> Pro rules.</a></li><li><strong>Community:</strong> Authored by Semgrep, Inc or external contributors such as Trail of Bits.</li><li><strong>Custom:</strong> Rules created within your Semgrep organization. For more information, see <a href="/writing-rules/private-rules/"><i class="fa-regular fa-file-lines"></i> Private rules</a>.</li></ul>.      |
+| Ruleset  | Rules are also organized in rulesets. Rulesets are groups of rules related through a programming language, OWASP category, or framework.       | <ul><li><a href="https://semgrep.dev/p/comment"><i class="fas fa-external-link fa-xs"></i> Comment ruleset</a></li><li><a href="https://semgrep.dev/p/react"><i class="fas fa-external-link fa-xs"></i> React ruleset</a></li></ul>       |
```

Comment:
This is sort of a weird one because one of the examples (comment) is not related through any of those categories, it's just rules we think are good for comment mode. I'm not really sure how to articulate that as a criterion, though.

Reply:
I'll change the example.